src/pages/pet/pet.component.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useEffect, Fragment } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { fetchPetProfile } from '../../store/actions/pet.action';
+import { fetchPetProfile } from '../../store/pet/action/pet.action';
```

Comment:
Name `action` folder in plural form. I think it will contains more than one action >_<

Reply:
done